packages/file/index.js

```diff
@@ -24,8 +45,22 @@ function gzipSize (path) {
 
 let self = {
   name: '@size-limit/file',
-  async step60 (config, check) {
+  async step60 (config, check, nodeVersion = process.version) {
```

Comment:
steps API do not support passing `nodeVersion` and it is not a big reason to add this argument to steps API.

It will be better to mock `process.version` in specific test.

Reply:
fixed